### DISCUSSION TITLE
error: cannot find symbol @With(ABC.class)

### DISCUSSION THREAD

**Discussion by yashmehta1** [playframework/playframework#11397] [Q&A] [answered]
Hi, 

This is the error I'm getting when i try to create a custom annotation. @With annotation is used for a custom annotation action class.

error: cannot find symbol @With(ABC.class)

What is the import module that I should use to resolve this issue ?

**Comment by mkurz** [comment, marked as answer]
I guess you made a type, the correct annotation is `@With`, with a capital W, see https://www.playframework.com/documentation/2.8.x/api/java/play/mvc/With.html
